js/main.js

```diff
@@ -114,61 +140,60 @@
 
       }
     ])
-    .service('true', ['$q',
-      function (q) {
-        return {
-          'get': function () {
-            var deferred = q.defer();
-
-            deferred.resolve(true);
+    .service('mockInterruptService', ['$q', '$log',
+      function (q, log) {
 
-            return deferred.promise;
-          }
+        function promise(value) {
+          var deferred = q.defer();
+          deferred.resolve(value);
+          return deferred.promise;
         }
-      }
-    ])
-    .service('false', ['$q',
-      function (q) {
+
         return {
-          'get': function () {
-            var deferred = q.defer();
 
-            deferred.resolve(false);
+          isInterruptRequired: function (interruptType) {
+            if (location.search.indexOf(interruptType) >= 0) {
+              return promise(true);
+            } else {
+              return promise(false);
```

Comment:
It's doing the right thing. The idea is to communicate 'no, this person definitely does not need to be interrupted' rather than 'something went wrong; we don't know if they should be interrupted or not'.